src/components/WriteButton/WriteButton.test.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import WriteButton from "./WriteButton";
+
+describe("WriteButton", () => {
+  test("Should be a button", () => {
```

Comment:
usually this starts with a lower case. :nail_care:

Reply:
Regardint the specs, I usually prefer something like:

```js
describe("<WriteButton />", () => {
  it("renders a button", () => {
    //...
  });
});
```
The test file becomes more readable and we are asserting and not expecting something.

```
returns -> It will return that (assertion)
```

```
should return -> yeah, maybe let's see (expectation)
```


then when running the specs it will display:
```
<WriteButton /> renders a button
```

way more readable too.

Reply:
Thanks! Nice feedback btw